Sources/XcodeGenCLI/Commands/GenerateCommand.swift

```diff
@@ -22,6 +22,9 @@ class GenerateCommand: ProjectCommand {
 
     @Flag("--only-plists", description: "Generate only plist files")
     var onlyPlists: Bool
+    
+    @Flag("--render-markdowns", description: "Render markdown files with `.md` extension")
+    var renderMarkdowns: Bool
```

Comment:
Rather than being flag to the generate command, I think it would be better if it was an option that could be set in the project spec. That's were all the other options for modifying the generated project are set.

Reply:
I have added this to the spec options too. (check that out please). But the flag is for the CLI, because the render is very likely to override by the developer but should not be changed from the spec options.

Reply:
@CraigSiemens Can you review it again? Thanks.